ex2_services/Services/source/housekeeping/housekeeping_service.c

```diff
@@ -227,15 +228,13 @@ Result mock_everyone(All_systems_housekeeping *all_hk_data) {
     all_hk_data->adcs_hk.Rate_Sensor_Temp_Z = tempFloat;
 
     // Athena
-    uint8_t i;
-    for (i = 0; i < ATHENA_TEMP_ARRAY_SIZE; i++) {
-        all_hk_data->Athena_hk.temparray[i] = tempLong;
-    }
+    all_hk_data->Athena_hk.MCU_core_temp = tempLong;
+    all_hk_data->Athena_hk.converter_temp = tempLong;
     all_hk_data->Athena_hk.boot_cnt = tempu16;
     all_hk_data->Athena_hk.OBC_mode = tempu8;
     all_hk_data->Athena_hk.OBC_uptime = tempu16;
     all_hk_data->Athena_hk.solar_panel_supply_curr = tempu8;
-    all_hk_data->Athena_hk.OBC_software_ver = tempu8;
+    memset(all_hk_data->Athena_hk.OBC_software_ver, 0, 8 * sizeof(char));
```

Comment:
Suggest having a variable or macro for 8*sizeof(char) titled something like "version_id_size" for clarity